main/res/values/strings.xml

```diff
@@ -747,23 +747,23 @@
     <string name="init_debug_note">c:geo can generate a lot of debugging information. Activating this setting will affect the stability of c:geo and is only recommended if you are asked to send a log to the developers. Please keep debug logging disabled during normal usage of c:geo!</string>
     <string name="init_debug">Activate debug log</string>
     <string name="init_dbonsdcard_title">Database location</string>
-    <string name="init_dbonsdcard_note">You may store the database of c:geo either system internally or in a directory on your user storage. Having it on the user storage may free some memory, but you may lose a bit of performance and c:geo will not work if the storage is not accessible or you remove the database file.</string>
+    <string name="init_dbonsdcard_note">You may store the database of c:geo either system internally or in a folder on your user storage. Having it on the user storage may free some memory, but you may lose a bit of performance and c:geo will not work if the storage is not accessible or you remove the database file.</string>
```

Comment:
Maybe a better wording would be necessary here. I really do not understand the text.

Reply:
That was IMHO targeted for older Android versions, where system internal and (internal) user storage had different quota or one of them could even be an SD-card. Nowadays on more recent Android version it makes no difference most of the time, but there might be special setups out there on some devices.
Therefore the text was explicitly made rather unspecific.

Reply:
The key sounds "db-on-sdcard" - is with `user storage` the storage on a additional sdcard meant?

Reply:
In android 1-3 there was no user storage directly on the device, just the private app-specific one. Later on, internal memory was added as an emulated SD card. So probably it has historical reasons... 

Nowadays you can choose between internal (emulated) memory and SD card (I stumbled across this setting a few days ago). The setting is useful, if you have a large db and prefer to use the SD card rather than the device memory

Reply:
I simplified this option. AFAIK its nowadays only relevant if you want to have the possibility to access the DB by yourself.